app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-      package="de.selfnet.wifisetup" android:installLocation="auto" android:versionCode="3" android:versionName="1.1-1">
+      package="de.selfnet.wifisetup" android:installLocation="auto" android:versionCode="4" android:versionName="1.1.2">
```

Comment:
```suggestion
      package="de.selfnet.wifisetup" android:installLocation="auto" android:versionCode="4" android:versionName="1.1-2">
```
I thought we would use the scheme MAJOR.MINOR-PATCH (at least we used to...).

Reply:
Personally `-PATCH` was also my preferred way.

Unfortunately other tooling like f-droid does not like it that much. 

In Issue #5 I linked to https://semver.org/ as this was what was recommended by the folks in the f-droid Issue-Tracker. (That is why I first made the Issue #5 and then created a merge request after you replied to it. :D )

Personally I would like to merge as is as then it would comply to semantic versioning standards foo. :)